bot/tests/not_sending_videos/test_delete_clip.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestDeleteClipCommand(BaseTest):
+
+    @pytest.mark.quick
+    def test_delete_existing_clip(self):
+        self.send_command('/klip cytat')
+        save_response = self.send_command('/zapisz test_clip')
+        save_expected_fragments = ["✅ Klip 'test_clip' został zapisany pomyślnie. ✅"]
+        self.assert_response_contains(save_response, save_expected_fragments)
+
+        delete_response = self.send_command('/usunklip 1')
```

Comment:
skąd wiesz że dodany ma numer 1? trochę dziwne jest kasowanie po numerku a nie nazwie tbh

Reply:
nawet nie że się czepiam testu tylko designu samego handlera który tera` wyszedł

Reply:
Wiem bo numer 1 zawsze jest na poczekaj a ta lista zawsze się od jedynki zaczyna czyli jak będzie 100 klipów i sto razy usune klip numer 1 to usune wszystkie klipy

Reply:
I była opcja po nazwie i teraz są chyba obie czy tam jedna nie działa xD w każdym razie przepisywanie nazw bywa frustrujące jak trzeba co do literki i polskiego znaku żeby zadziałało

Reply:
> przepisywanie nazw bywa frustrujące

I dlatego mowilem zebys wyciagnal do zmiennej a potem podstawial do stringa 😛

Reply:
a używać tych response od bota w sensie tych funkcji zwracających strigny w testach czy testy powinny być od tego niezależne ?

Reply:
yep, dokladnie o to chodzi